.github/workflows/label_apply_change.yml

```diff
@@ -58,6 +103,9 @@ jobs:
         run: |
           echo Label exists
           echo rename: ${{ inputs.rename }}
+          # TODO check if we actually have to treat the boolean as string here
+          # See https://docs.github.com/en/actions/learn-github-actions/contexts#inputs-context
+          # See https://github.com/actions/runner/issues/1483
           if [ ${{ inputs.rename }}  == 'true' ];then
```

Comment:
Impressive digging! 🕵️ This was interesting to read up on.

Some of that issue thread you linked is still a little over my head, but based on [this](https://github.com/actions/runner/issues/1483#issuecomment-1157967945) comment from a contributor it does sound like the if statement _should_ work without `== 'true'` part...not sure if you've tested that out already? 

[This](https://github.com/actions/runner/issues/1483#issuecomment-1169304583) comment and the corresponding [reply](https://github.com/actions/runner/issues/1483#issuecomment-1169313156) also seems relevant to how we're doing things here. Maybe you've seen these already. 🙂